flowmap/cli.py

```diff
@@ -197,21 +197,26 @@ def streamlines(dataset, timestep, **kwargs):
 )
 @click.option(
     "--method",
-    type=click.Choice(['subgrid', 'interpolate']),
-    default="interpolate"
+    type=click.Choice(['waterdepth', 'waterlevel', 'interpolate']),
+    default="waterlevel"
+)
+@click.option(
+    "--format",
+    type=click.Choice(['.geojson', '.tiff']),
```

Comment:
I think its weird to choose between extensions here, I would expect a geojson or tiff here